source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -18,11 +18,9 @@ than BSON documents. Because these methods follow the fluent interface
 pattern, you can chain aggregation operations together to create code
 that is both more compact and more naturally readable.
 
-.. tip:: 
-
-   For more information about aggregation expressions that use the Query API, see the 
-   :manual:`Aggregation Pipeline Operators </reference/operator/aggregation/>` 
-   page in the Server manual.
+The methods in this guide provide an idiomatic way to implement the Query API,
+a framework that you can use to interact with your data. To learn more
```

Comment:
Suggestion:

I'm not sure there are any other ways to interact with MongoDB, so maybe narrowing it down could make this more succinct:

```suggestion
the framework that lets the driver interact with MongoDB. To learn more
```